.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           vid: ${{ secrets.VERACODE_API_ID }}
           vkey: ${{ secrets.VERACODE_API_KEY }}
           file: "distribution/target/alfresco-hxinsight-connector-distribution-*.zip"
-          fail_build: true
+          fail_build: false
```

Comment:
Thanks for doing this. As discussed we already have automated Jira tickets created whenever there is a new issue found in a library by Veracode. We don't want to fail our builds whenever this happens.